src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -839,9 +827,9 @@ int mongoSubCacheItemInsert(const char* tenant, const BSONObj& sub, const char*
   std::vector<BSONElement>  attrVec       = sub.getField(CSUB_ATTRS).Array();
   std::vector<BSONElement>  condVec       = sub.getField(CSUB_CONDITIONS).Array();
 
-  if ((lastNotificationTime == -1) && (sub.hasField(CSUB_LASTNOTIFICATION)))
+  if (lastNotificationTime == -1)
```

Comment:
sub.hasField was removed at is was done inside tghe old version of getLongField.
Now this has changed. hasField must come back here too.

Good catch.  However, let's take a look at the entire function ...